### PR TITLE
Addresses #1579 [NTR] [ASCT+B table] descending thin limb epithelial …

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Addresses #1579 [NTR] [ASCT+B table] descending thin limb epithelial cell (3 types)

### CODE REVIEW

**Review by paolaroncaglia — COMMENTED** [review]
@bvarner-ebi this looks ok to me, but I know that you may potentially add some definition gloss or comments about markers if @emquardokus hears back from the expert, as per yesterday's call. Feel free to ask for my review again.

**Comment by bvarner-ebi** [comment]
@emquardokus, could you kindly review my proposed updates? 
FYI, @paolaroncaglia

**Comment by bvarner-ebi** [comment]
@emquardokus, per yesterday's discussion, we agreed that references to biomarkers can be relegated to definition gloss (i.e., not appearing in the first sentence of the text definitions) unless there is a compelling reason.

@paolaroncaglia, if there is no objection by end of day today, would you review and approve (if appropriate) this PR so I can move forward with adding the other kidney cell terms?

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -30104,6 +30107,36 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030011 "epithelial cell of
 AnnotationAssertion(rdfs:label obo:CL_4030011 "epithelial cell of proximal tubule segment 3")
 SubClassOf(obo:CL_4030011 obo:CL_0002306)
 
+# Class: obo:CL_4030012 (kidney loop of Henle short descending thin limb epithelial cell)
+
+AnnotationAssertion(Annotation(<http://purl.org/dc/elements/1.1/contributor> <http://orcid.org/0000-0003-2804-127X>) Annotation(oboInOwl:hasDbXref "PMID:33769951"^^xsd:string) obo:IAO_0000115 obo:CL_4030012 "Epithelial cell of the descending thin limb of the short loop (cortical) nephron limited to the outer medulla (mainly inner strip). It is known in some mammalian species that the short descending limb of the loop of Henle selectively expresses the serine protease Corin, the homeobox TF Uncx, and the urea channel Slc14a2."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030012 <http://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030012 "2022-05-13T11:00:29Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:editor_note obo:CL_4030012 "TO DO: import UBERON:0005099 'short descending thin limb' and add subClass: part_of some UBERON:0005099"^^xsd:string)
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:30115]: @bvarner-ebi I'd rather move this action item to a ticket, unless you're confident that you'll get to it soon, and that you'll remember to remove the editor note promptly ;-) But up to you really.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -30104,6 +30107,36 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030011 "epithelial cell of
...
+SubClassOf(obo:CL_4030012 obo:CL_1001111)
+
+# Class: obo:CL_4030013 (kidney loop of Henle long descending thin limb outer medulla epithelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <http://orcid.org/0000-0003-2804-127X>) Annotation(oboInOwl:hasDbXref "PMID:33769951"^^xsd:string) obo:IAO_0000115 obo:CL_4030013 "Epithelial cell of the descending thin limb of the long loop (juxtamedullary) nephron that spans the outer medulla (inner stripe). It is known in some mammalian species that the long descending limb of the loop of Henle in the outer medulla selectively expresses the secreted activin-antagonist protein follistatin (Fst), the GPI-linked adhesion protein Cdh13, and the protein kinase Stk32a."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030013 <http://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030013 "2022-05-13T11:11:07Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:editor_note obo:CL_4030013 "TO DO: import UBERON:0005100 'long descending thin limb' and add subClass: part_of some UBERON:0005100"^^xsd:string)
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:30125]: See above

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -30104,6 +30107,36 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030011 "epithelial cell of
 AnnotationAssertion(rdfs:label obo:CL_4030011 "epithelial cell of proximal tubule segment 3")
 SubClassOf(obo:CL_4030011 obo:CL_0002306)
 
+# Class: obo:CL_4030012 (kidney loop of Henle short descending thin limb epithelial cell)
+
+AnnotationAssertion(Annotation(<http://purl.org/dc/elements/1.1/contributor> <http://orcid.org/0000-0003-2804-127X>) Annotation(oboInOwl:hasDbXref "PMID:33769951"^^xsd:string) obo:IAO_0000115 obo:CL_4030012 "Epithelial cell of the descending thin limb of the short loop (cortical) nephron limited to the outer medulla (mainly inner strip). It is known in some mammalian species that the short descending limb of the loop of Henle selectively expresses the serine protease Corin, the homeobox TF Uncx, and the urea channel Slc14a2."^^xsd:string)
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:30112]: @bvarner-ebi looks like you'll need to run the script to remove xsd:strings, there are several in this PR. Thank you.

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -30104,6 +30107,36 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030011 "epithelial cell of
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <http://orcid.org/0000-0003-2804-127X>) Annotation(oboInOwl:hasDbXref "PMID:33769951"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/gene/8170"^^xsd:string) obo:IAO_0000115 obo:CL_4030014 "Epithelial cell of the descending thin limb of the long loop (juxtamedullary) nephron that spans the inner medulla. It is known in some mammalian species that the long descending limb of the loop of Henle in the inner medulla selectively expresses the nuclear receptor Nr2e3, the Ig kappa chain Igkc, and the secreted protein dermokine (Dmkn). SLC14A2, which expresses a urea transporter, is also expressed in the inner medulla."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030014 <http://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030014 "2022-05-13T11:16:54Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:editor_note obo:CL_4030014 "TO DO: import UBERON:0005100 'long descending thin limb' and add subClass: part_of some UBERON:0005100"^^xsd:string)
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:30135]: See above

**Review by paolaroncaglia — COMMENTED** [review]
@bvarner-ebi once my suggestions are addressed (some are optional), I'm good with this PR. I'm commenting now because I'll be offline the rest of today, and Monday morning too, most likely.

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:30112]: I thought I did run it... I don't think it applies to the newly added terms but could be wrong. I will run it again though.

File: src/ontology/cl-edit.owl (hunk 4 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:30135]: #1605 created; editor notes removed.

**Review by paolaroncaglia — APPROVED** [review]

**Pull request merged**